### ISSUE TITLE
[from-map] Does not work with TS files

### ISSUE THREAD

**Issue by doniyor2109** [freaktechnik/eslint-plugin-array-func#49]
#### Config
```json
{
  "parserOptions": {
    "sourceType": "module"
  },
  "parser": "@typescript-eslint/parser",
  "plugins": ["array-func"],
  "rules": {
    "array-func/from-map": "error"
  }
}
```

#### Code 
```ts
Array.from({length: 5 }).map((x) => x);
```

#### Expected behavior
Should fix correctly as following:

```ts
Array.from({length: 5 }, (x) => x);
```

#### Actual behavior
Fixes incorrect:
```ts
, Array.from({length: 5 }).map((x) => x);
```

Here is repo: https://github.com/doniyor2109/demo-error-for-array-func-from-map

**Comment by freaktechnik** [comment]
Hm, intuitively I'd expect it to work fine, though there may be subtle differences in the AST the typescript parser produces. Luckily AST explorer seems to support the typescript parser.

**Comment by doniyor2109** [comment]
@freaktechnik Yea, it detects the rule but fixing incorrectly.

**Comment by freaktechnik** [comment]
Fixed as of version 3.1.5 :tada:

**Comment by doniyor2109** [comment]
@freaktechnik Thanks.
